resources/widgets/EntityAutocompleteWidget.less

```diff
@@ -0,0 +1,45 @@
+@import '../style-variables.less';
+
+.wbmad-entity-input {
+	max-width: 100%;
+
+	input {
+		text-overflow: ellipsis;
+	}
+
+	/* stylelint-disable-next-line selector-class-pattern */
+	&.oo-ui-flaggedElement-destructive {
+		input {
+			color: #d33;
+			border-color: #d33;
+		}
+	}
+}
+
+.wbmad-entity-input__autocomplete__label,
+.wbmad-entity-input__autocomplete__description,
+.wbmad-entity-input__autocomplete__aliases {
+	color: @base10;
+	overflow: hidden;
```

Comment:
This line and the 3 following it can all be removed and replaced with `.text-overflow( @visible: false )` if you import mediawiki mixins to this file (see https://gerrit.wikimedia.org/r/c/mediawiki/extensions/MachineVision/+/592432/3/resources/widgets/EntityAutocompleteInputWidget.less)